.circleci/config.base.yml

```
@@ -114,6 +114,26 @@ jobs:
           no_output_timeout: 90m
       - store_test_results:
           path: packages/graphql-transformers-e2e-tests/
+  amplify_sudo_install_test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ./
+      - restore_cache:
+          key: amplify-verdaccio-cache-{{ .Branch }}-{{ .Revision }}
+      - run:
+          name: Update OS Packages
+          command: sudo apt-get update
+      - run:
+          name: Start verdaccio and Install Amplify CLI as sudo
+          command: |
+            source .circleci/local_publish_helpers.sh
+            startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
+            setSudoNpmRegistryUrlToLocal
```

can we run the same command with sudo
```suggestion
            sudo setNpmRegistryUrlToLocal
```

---

tried it out and that doesn't seem to work: `sudo: setNpmRegistryUrlToLocal: command not found`. Also tried `sudo source .circleci/local_publish_helpers.sh` before and that also failed with `sudo: source: command not found`